GitVersionConfig.yaml

```diff
@@ -1,3 +1,3 @@
 mode: ContinuousDelivery
-next-version: 2.1.0
+next-version: 3.0.0

```

Comment:
Major version bump - if someone used flags enum previously then it will now return different HTML.